_data/transport.yml

```diff
@@ -222,6 +222,14 @@ websites:
     facebook: transportforlondon
     img: tfl.png
 
+  - name: TripIt
+    url: https://www.tripit.com
+    twitter: TripIt
+    facebook: tripitcom
+    email_address: security@tripit.com
+    img: tripit.png
+    exception: "Send a more secure email using PGP key downloaded from https://tripit.com/pgp/TripIt_Security_PUB.asc."
```

Comment:
That’s not a 2FA implementation exception.
```suggestion
```

Reply:
Jekyl failed when I tried using `doc`, should the Jekyl test account for an exception without a 2FA implementation?**(bug/enhancement)**

Where/how can the information regarding the PGP key be listed in regards to the email found https://www.tripit.com/web/lp/security/#report

Reply:
I have committed the suggested change.